runing/routers.py

```
@@ -0,0 +1,51 @@
+from servuces import UserService
+
+router = APIRouter(prefix="/users", tags =["users"])
+
+def get_user_service(db: Session = Depends(get_db)) -> UserService:
+    user_repository = UserRepository(db)
+    return UserService(user_repository)
+
+@router.post("/", response_model=schemas.user)
+def create_user(
+    user: schemas,UserCreate,
+    user_service: UserService = Depends(get_user_service)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)